src/GwentTracker/locale/cards.ru.po

```diff
@@ -485,7 +485,7 @@ msgstr "Я могу стать твоей последней - и самой л
 
 #: src/GwentTracker/data/monsters.yml:488 -> Card.name
 msgid "Eredin King of the Wild Hunt"
-msgstr "Эредин Бреакк Глас Король Aen Elle"
+msgstr "Эредин Бреакк Глас Командир Дикой Охоты"
```

Comment:
I think the Eredin card titles got mixed up with each other. Lines 488 with 500 and 512 with 520.

Reply:
Yes, sorry, I checked the names in the old card base.